casper/src/main/scala/coop/rchain/casper/engine/Running.scala

```diff
@@ -51,42 +48,33 @@ object Running {
     * To mitigate this issue we can update fork choice tips if current fork-choice tip has old timestamp,
     * which means node does not propose new blocks and no new blocks were received recently.
     */
-  def updateForkChoiceTipsIfStuck[F[_]: Sync: CommUtil: Log: Time: BlockStore: BlockDagStorage: EngineCell](
+  def updateForkChoiceTipsIfStuck[F[_]: Sync: CommUtil: Log: Time: BlockStore: BlockDagStorage](
       delayThreshold: FiniteDuration
   ): F[Unit] =
     for {
-      engine <- EngineCell[F].read
-      _ <- engine.withCasper(
```

Comment:
Now `withCasper` is used only in tests. Can we fully remove it?

Reply:
Good catch. I've added a few more commits and removed Casper interface completely. :partying_face: :balloon: